vtex/loaders/product/extend.ts

```diff
@@ -16,11 +16,8 @@ export interface Props {
   products: Product[];
 }
 
-const similarsExt = (
-  products: Product[],
-  req: Request,
-  ctx: AppContext,
-) => Promise.all(products.map((p) => withIsSimilarTo(req, ctx, p)));
+const similarsExt = (products: Product[], req: Request, ctx: AppContext) =>
```

Comment:
There were a lot of code formatting changes

Could you double-check that your editor's format is not overriding Deno's default?

https://docs.deno.com/runtime/manual/tools/formatter